crates/bevy_app/src/ci_testing.rs

```diff
@@ -20,18 +22,29 @@ fn ci_testing_exit_after(
     if let Some(exit_after) = ci_testing_config.exit_after {
         if *current_frame > exit_after {
             app_exit_events.send(AppExit);
+            info!("Exiting after {} frames. Test successful!", exit_after);
```

Comment:
I wonder if we should be setting some custom property on `window` instead. That way we're not relying on the fairly large number of assumptions that mean `info` lands in the console.

Reply:
if you think the other way around, this is also a test that logs lands in the console